Basic/02_tipe_data/tipe_data.py

```diff
@@ -8,8 +8,11 @@
 
 # contoh tipe data str
 teks_string = "Halo dunia!"
+teks_string_2 = 'halo dunia!'
```

Comment:
string pada python disarankan menggunakan double quote
```suggestion
teks_string_2 = "halo dunia!"
```